### PR TITLE
Feature/home page

### PR DESCRIPTION
**Pull request by GabLeRoux**
Some landing page. hell less, sass, css, next, antd, this is a nightmare, used to be easier back in the days.

- [ ] Add `less` support to allow antd customization (sounds easier than it really is)
- [x] Remove home redirection to documentation
- [ ] Add a call to action to direct to the right place in documentations

### CODE REVIEW

**Comment by GabLeRoux** [comment]
Wow, I managed to fail vercel deployment

File: components/layout/components/nav-bar.js [hunk 1]
```diff
@@ -1,128 +1,139 @@
...
-  </Header>
-);
+const NavBar = () => {
+  const { asPath: currentKey } = useRouter();
+  let defaultSelectedKeys = '';
+  if (currentKey === '/') {
+    defaultSelectedKeys = '/';
+  } else if (currentKey.includes('/docs')) {
+    defaultSelectedKeys = '/docs';
+  }
```
Comment (webbertakken) [inline components/layout/components/nav-bar.js:18]: Please note that my pr also rebuilds almost everything in this component, due to search becoming part of it in #73.

**Comment by webbertakken** [comment]
Overall looks great! Hopefully we're able to resolve the conflicting style loaders. This would be a great addition to the website!

**Comment by webbertakken** [comment]
Shall we say this is superseeded by #162 ?

**Comment by GabLeRoux** [comment]
For sure this is definitely outdated. We'll aim for a more simple front page for now 🙌

**Pull request closed without merging**